opennms-webapp/src/main/webapp/admin/sched-outages/editoutage.jsp

```diff
@@ -158,7 +158,8 @@
 		}
 	}%>
 <%
-    NotifdConfigFactory.init(); //Must do this early on - if it fails, then just throw the exception to the web gui
+    NotifdConfigFactory.init();
+    NotifdConfigFactory.getInstance().reload(); //Must do this early on - if it fails, then just throw the exception to the web gui
```

Comment:
is that possible to not call reload here ? (your init reload is removed, and postConstruct did not work properly?)